packaging/linux/flatpak/scripts/install-user-service.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+mkdir -p ~/.config/systemd
+mkdir -p ~/.config/systemd/user
+cp /app/share/sunshine/systemd/user/sunshine.service $HOME/.config/systemd/user/sunshine.service
+sed -i 's/\/app\/bin\/sunshine/flatpak run dev.lizardbyte.sunshine\nExecStop=flatpak kill dev.lizardbyte.sunshine/g' $HOME/.config/systemd/user/sunshine.service
```

Comment:
You can use another symbol to separate the `sed` command, so you don't need to escape so much.  You can use any symbol you like.
Example:

```bash
sed -i 's%/app/bin/sunshine%flatpak run dev.lizardbyte.sunshine\nExecStop=flatpak kill dev.lizardbyte.sunshine%g' $HOME/.config/systemd/user/sunshine.service
```

I think we should move the `sed` command to the manifest. If a user runs the `install-user-service.sh` script more than once it could have unintended results.

Also, has the `sed` command been tested?